py/client/README.md

```diff
@@ -62,6 +62,202 @@ $ pip3 install dist/pydeephaven-0.12.0-py3-none-any.whl
 
 ```
 
+## Initialize
+
+The `Session` class is your connection to Deephaven. This is what allows your Python code to interact with a Deephaven server.
+
+```
+from pydeephaven import Session
+
+session = Session()
+```
+
+## Ticking table
+
+The `Session` class has many methods that create tables. This example creates a ticking time table and binds it to Deephaven.
+
+```
+from pydeephaven import Session
+
+session = Session()
+
+table = session.time_table(period=1000000000).update(formulas=["Col1 = i % 2"])
+session.bind_table(name="my_table", table=table)
+```
+
+This is the general flow of how the Python client interacts with Deephaven. You create a table (new or existing), execute some operations on it, and then bind it to Deephaven. Binding the table gives it a named reference on the Deephaven server.
```

Comment:
Might want to give the motivation for why a named reference is significant.

"... gives it a named reference on the Deephaven server, so that it can be used from the Web API or other Sessions."

Reply:
I like it, done